### PR TITLE
Optimize serialized server/client messages

### PR DESCRIPTION
**Pull request by satyaog**
# Description

Optimize the serialization of the messages sent between the simulation and the inference server.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] python run_tests.py

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/covid19sim/frozen/utils.py [hunk 1]
```diff
@@ -1,46 +1,30 @@
...
-    received_at = float(received_at)  # datetime.datetime.strptime(received_at, "%Y-%m-%d %H:%M:%S")
-    unobs_uid = unobs_id
-    return UpdateMessage(obs_uid, new_risk, risk, day, received_at, unobs_uid)
+    return UpdateMessage(*update_message)
 
 
 def create_new_uid(rng):
     # generate a 4 bit random code
-    return np.random.randint(0, 15)
+    return rng.randint(0, 15)
```
Comment (obilaniu) [inline src/covid19sim/frozen/utils.py]: `np.random.randint(lo, hi)` has inclusive `lo` but exclusive `hi`, so this code will never generate UID `15`.

Comment (satyaog) [reply src/covid19sim/frozen/utils.py]: Thanks I didn't realized

**Comment by plstcharles** [comment]
i'm late on all sorts of PR reviews, and I just discovered this one --- are we sure these changes won't cause issues with upcoming changes elsewhere (experimental/clustering/human/other stuff?)

**Review by plstcharles — APPROVED** [review]
meh, it's not too bad, if there are fixes needed on the clustering side I can solve those

**Pull request merged**